src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -60,6 +60,8 @@ internal LuceneSearchUtility(DynamoModel model)
         /// </summary>
         internal void InitializeLuceneConfig(string dirName, LuceneStorage storageType = LuceneStorage.FILE_SYSTEM)
         {
+            if (DynamoModel.IsHeadless) return;
```

Comment:
So this should be set from all CLIs and D4R when the model starts without UI right ? 
Also we should check with D4C3D.

Reply:
I think so